app/src/main/java/com/celzero/bravedns/util/OrbotHelper.kt

```diff
@@ -274,6 +278,7 @@ class OrbotHelper(private val persistentState: PersistentState, private val prox
         val contentText = context.resources.getString(R.string.settings_orbot_notification_content)
         builder.setSmallIcon(R.drawable.dns_icon).setContentTitle(contentTitle).setContentIntent(mainActivityIntent).setContentText(contentText)
         builder.setStyle(NotificationCompat.BigTextStyle().bigText(contentText))
+        builder.color = ContextCompat.getColor(context, fetchColor(context))
```

Comment:
notification should probably just follow system-theme and not app-theme?

Reply:
Notification now will follow the system-theme.